middleware/throttle.go

```diff
@@ -2,23 +2,43 @@ package middleware
 
 import (
 	"net/http"
+	"time"
 
 	"github.com/pressly/chi"
 	"golang.org/x/net/context"
 )
 
-// Throttle is a middleware that limits number of currently
-// processed requests at a time.
-func Throttle(limit int) func(chi.Handler) chi.Handler {
-	if limit <= 0 {
+var (
+	defaultTimeout = time.Second * 60
```

Comment:
is this the backlog timeout? I think then it should be named `defaultBacklogTimeout`